code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -194,6 +194,7 @@
 	description = "If used in touch-based applications, immediately restores burn wounds as well as restoring more over time. If ingested through other means, deals minor toxin damage."
```

Comment:
All descriptions need updating unless you mean for this to be only detectable via chemmaster and codereading which is dumb.

Reply:
Chem master only